app.js

```diff
@@ -0,0 +1,29 @@
+require("dotenv").config;
+
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const routes = require("./routes");
+
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(cors());
+app.use(morgan("combined"));
+app.use(express.json());
+app.use(routes);
+
+app.get("/ping", (res, req) => {
+  res.statusCode(200).json({ message: err.message });
+}); // health check
```

Comment:
```suggestion
});
```
- 불필요한 주석은 삭제해주세요!
- 코드에 대한 설명을 주석으로 남겨놓고 싶어서 주석을 달게 되는 것은 처음에는 좋아보일 수는 있으나 나중에 코드에 수정이 발생했을 때 설명하는 주석까지 같이 수정하는 것을 놓치게 된다면 결국 설명 주석과 코드의 내용이 달라지고 오히려 더 유지보수하기 힘들게 될 수 있습니다.
- 주석으로 코드를 설명하기보다는 코드 자체의 가독성이 좋게만들어서 부가적인 설명이 필요없는 코드로 만들어주세요!